src/pages/installing/upgrading.md

```diff
@@ -1,26 +1,26 @@
 ---
 title: "Upgrading"
-excerpt: "Upgrading an ACD installation."
+excerpt: "Upgrading an IBM Watson ACD installation."
 categories: installing
 slug: upgrading
 toc: true
 ---
-The Annotator for Clinical Data Container Edition supports both manual and automatic upgrades of the operator and operand through [Operator Lifecycle Manager (OLM)](https://docs.openshift.com/container-platform/4.7/operators/understanding/olm/olm-understanding-olm.html#olm-overview_olm-understanding-olm) integration. OLM operators are limited to upgrades and do not rollback function. An operator upgrade is always done with a new replacement version. Operand upgrades are managed by the operator. An ACD instance is upgraded based on a specific set of related images that align with the ACD operator version.
 
-## Upgrade Paths
+_Note:  The upgrade path only applies to IBM Watson Annotator for Clinical Data Container Edition._
```

Comment:
Add something like the following to clarify ... The upgrade path only applies to IBM Watson Annotator for Clinical Data Container Edition at this time. Upgrades will also be available for Merative Annotator for Clinical Data Container Edition sometime after the initial Merative version is published.